libs/ui/src/components/MediaItem/type/ImageMedia.vue

```diff
@@ -30,6 +32,7 @@ const onError = (e: Event) => {
   const target = e.target as HTMLImageElement
   if (target) {
     consola.log('[KODADOT::IMAGE] unable to load', props.src, e)
+    target.removeAttribute('srcset')
     target.src = props.placeholder
   }
```

Comment:
Please replace these parts with `placeholder` if they are doing the same thing
`https://image.nuxt.com/usage/nuxt-img#placeholder`

Reply:
No, it's not the same thing, nuxt-image placeholder is for showing placeholder during loading image

Reply:
Ok. The second question is why we should add this line`target.removeAttribute('srcset')`

Reply:
Because we need to replace src to placeholder, but NuxtImg set srcset attribute so browser takes image from srcset, that's why we have to clear it